ucrop/src/main/java/com/yalantis/ucrop/util/BitmapLoadUtils.java

```diff
@@ -24,6 +25,7 @@
     private static final String TAG = "BitmapLoadUtils";
 
     @Nullable
+    @TargetApi(Build.VERSION_CODES.JELLY_BEAN)
```

Comment:
Add to do this in order to get lint to ignore the casting of the `ParcelFileDescriptor` as an error to `Closeable`.